examples/schema/curation_rule-propagated.sssom.tsv

```diff
@@ -0,0 +1,17 @@
+#curie_map:
+#  HP: http://purl.obolibrary.org/obo/HP_
+#  MP: http://purl.obolibrary.org/obo/MP_
+#  orcid: https://orcid.org/
+#  DISEASE_MAPPING_COMMONS_RULES: https://w3id.org/sssom/commons/disease/curation-rules/
+#mapping_set_id: https://w3id.org/sssom/commons/examples/curation_rule.sssom.tsv
+#mapping_set_description: "This example illustrates how to express that all mappings in a mapping set have been curated according to a specific curation rule"
+#license: "https://creativecommons.org/publicdomain/zero/1.0/"
+#creator_id: orcid:0000-0002-7356-1779
+#mapping_provider: "https://w3id.org/sssom/core_team"
+#curation_rule:
+#  - DISEASE_MAPPING_COMMONS_RULES:MPR2

```

Comment:
can you please use an example of a curation rule that can be resolved when expanded to a URI?

I think for most SSSOM curators/developers/users, it isn't clear what a curation rule is nor why you should use one, so it would be important that the example uses a curation rule that has some real context attached to it

Reply:
@cthoyt I’d say: outside of the scope of this PR.

If we were adding completely new slots, I could agree, but here, both `curation_rule` and `curation_rule_text` have existed for a while, this PR is solely about making them propagatable.

If you think that some existing slots do not have enough meaningful examples, I’d rather we open another issue and tackle that problem separately.

Reply:
Thanks @nichtich and @cthoyt -> @gouttegd and I moved this to an issue and will address this as a separate documenation effort: https://github.com/mapping-commons/sssom/issues/479